src/components/utils/parseGridTemplateAreas.ts

```diff
@@ -0,0 +1,3 @@
+export function parseGridTemplateAreas(rows: string[]): string[][] {
```

Comment:
- Combine similar utils into one file so this and `toGridTemplateAreas`
- import/export them into an index for easy finding since there are only a few?

Reply:
good idea. moved them to `utils/gridHelpers`